<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editFrame</class>
 <widget class="QDialog" name="editFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#background{
background-color: #a2b5b0;}</string>
   </property>
   <widget class="QWidget" name="waterWgt" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>500</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#waterWgt{border-radius: 15px;
background-color: #C5DCD6;}</string>
    </property>
    <widget class="QLineEdit" name="nameField">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f7f7f7;
font: &quot;Lato&quot;;
font-size: 22px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="varietyLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;</string>
     </property>
     <property name="text">
      <string>Variety</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="varietyField">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f7f7f7;
font: &quot;Lato&quot;;
font-size: 22px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="waterLabel1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>Water every</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="waterField">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f7f7f7;
font: &quot;Lato&quot;;
font-size: 22px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lastWateredLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>Last Watered</string>
     </property>
    </widget>
    <widget class="QLabel" name="waterLabel2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>days</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="lastWateredField">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>9999</year>
       <month>8</month>
       <day>30</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1752</year>
       <month>10</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="bdayLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>360</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>Birthday</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="bdayField">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>400</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>9999</year>
       <month>8</month>
       <day>30</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1752</year>
       <month>10</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="updateBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>500</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font: &quot;Lato&quot;;
border: 4px solid '#bbd1cb';
border-radius: 10px;
font-size: 20px;
background-color: #bbd1cb
}
*:hover{
border: 4px solid '#b4c8c2';
background-color:'#b4c8c2';
}
</string>
     </property>
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato Semibold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #cc0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>590</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font: &quot;Lato&quot;;
border: 4px solid '#d11a2a';
border-radius: 10px;
font-size: 20px;
background-color: #d11a2a;
color: white;
}
*:hover{
border: 4px solid '#f71f38';
background-color:'#f71f38';
}
</string>
     </property>
     <property name="text">
      <string>Delete Plant</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>81</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font: &quot;Lato&quot;;
font-size: 64px;
border-radius: 5px;
color: #ebebeb;
}
</string>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
